example/test/avoid_returning_widget_test.dart

```diff
@@ -0,0 +1,31 @@
+// ignore_for_file: unused_element
+
+/// Check returning a widget fail
+/// `avoid_returning_widgets`
+
+import 'package:flutter/widgets.dart';
+
+// expect_lint: avoid_returning_widgets
+Widget avoidReturningWidgets() => const SizedBox();
+
+class MyWidget extends StatelessWidget {
+  const MyWidget({super.key});
+
+  // expect_lint: avoid_returning_widgets
+  Widget _test1() => const SizedBox();
+
+  // expect_lint: avoid_returning_widgets
+  Widget _test2() {
```

Comment:
Does this produce a lint too?
```suggestion
  dynamic _test2() {
```

Reply:
Not in scope now. Right now the priority is to get off DCM, we should focus that.
This probably worked the same in DCM, also.